nym-wallet/src/pages/bonding/node-settings/NodeSettings.tsx

```diff
@@ -0,0 +1,147 @@
+  const { search } = useLocation();
+
+  return React.useMemo(() => new URLSearchParams(search), [search]);
+};
+
+const getTabs = () => {
+  const tabs = Object.values(nodeSettingsNav) as string[];
+  const length = tabs.length;
+  return tabs.splice(0, length / 2);
+};
+
+export const NodeSettings = () => {
+  const theme = useTheme();
+  const { network } = useContext(AppContext);
+  const { bondedNode, unbond, isLoading } = useBondingContext();
+  const navigate = useNavigate();
+  const query = useQuery();
+  const queryTab = query.get('tab');
+  const tabs = getTabs();
+
+  const [confirmationDetails, setConfirmationDetails] = useState<ConfirmationDetailProps>();
+  const [value, setValue] = React.useState(queryTab === 'unbond' ? 1 : 0);
+  const handleChange = (event: React.SyntheticEvent, tab: number) => {
+    setValue(tab);
+  };
+
+  const handleUnbond = async (fee?: FeeDetails) => {
+    const tx = await unbond(fee);
+    setConfirmationDetails({
+      status: 'success',
+      title: 'Unbond successful',
+      txUrl: `${urls(network).blockExplorer}/transaction/${tx?.transaction_hash}`,
+    });
+  };
+
+  const handleError = (error: string) => {
+    setConfirmationDetails({
+      status: 'error',
+      title: 'An error occurred',
+      subtitle: error,
+    });
+  };
+  return (
+    <PageLayout>
+      <NymCard
+        borderless
+        noPadding
+        title={
+          <Stack gap={2} sx={{ py: 0 }}>
+            <Box
+              sx={{
+                display: 'flex',
+                alignItems: 'center',
+                justifyContent: 'space-between',
+              }}
+            >
+              <Box sx={{ display: 'flex', alignItems: 'center', gap: 1 }}>
+                <NodeIcon />
+                <Typography variant="h6" fontWeight={600}>
+                  Node Settings
+                </Typography>
+              </Box>
+            </Box>
+            <Box sx={{ width: '100%' }}>
+              <Tabs
+                tabs={tabs}
+                selectedTab={value}
+                onChange={handleChange}
+                tabSx={{
+                  bgcolor: 'transparent',
+                  borderBottom: 'none',
+                  borderTop: 'none',
+                  '& button': {
+                    p: 0,
+                    mr: 4,
+                    minWidth: 'none',
+                    fontSize: 16,
+                  },
+                  '& button:hover': {
+                    color: theme.palette.nym.highlight,
+                    opacity: 1,
+                  },
+                }}
+                tabIndicatorStyles={{ height: 4, bottom: '6px', borderRadius: '2px' }}
+              />
+            </Box>
+          </Stack>
+        }
+        Action={
+          <Button
+            size="small"
+            sx={{
+              color: 'text.primary',
+            }}
+            onClick={() => navigate('/bonding')}
+            startIcon={<Close />}
+          ></Button>
```

Comment:
Change this button to an IconButton component and add <CloseIcon /> from mui icons as a child